great_expectations/core/validation_config.py

```diff
@@ -23,10 +20,25 @@ class ValidationConfig(BaseModel):
 
     """
 
+    class Config:
+        arbitrary_types_allowed = True
+        json_encoders = {
+            ExpectationSuite: lambda v: expectationSuiteSchema.dump(v),
+        }
+
     name: str
     data: BatchConfig  # TODO: Should support a union of Asset | BatchConfig
     suite: ExpectationSuite
 
+    @validator("suite", pre=True)
+    def _validate_suite(cls, v):
+        if isinstance(v, dict):
+            return ExpectationSuite(**expectationSuiteSchema.load(v))
+        return v
+
     @public_api
     def run(self):
         raise NotImplementedError
+
+
+ValidationConfig.update_forward_refs()
```

Comment:
Not sure why this is necessary 🤔

Reply:
hmm.. just to double check - any chance this was needed before they type checking block was removed, but is no longer required?

Reply:
I believe you're right!